benchmarks/definitions/nasbench201.py

```diff
@@ -13,7 +13,7 @@
 from pathlib import Path
```

Comment:
Would this file not move to benchmarking/definitions/ ?

Reply:
yes this would eventually move to benchmarking/definitions. To not clutter the PR too much I just moved exemplary one benchmark